code/modules/cargo/exports/large_objects.dm

```diff
@@ -130,8 +130,8 @@
 	worth += gases[/datum/gas/freon][MOLES]*15
 	worth += gases[/datum/gas/tritium][MOLES]*5
 	worth += gases[/datum/gas/pluoxium][MOLES]*5
-	worth += gases[/datum/gas/bz][MOLES]*4
-	worth += gases[/datum/gas/miasma][MOLES]*2
+	worth += gases[/datum/gas/bz][MOLES]*5
```

Comment:
I don't think this is a good idea

Reply:
and the negative co2 money is pretty meh, and it will allow the rise of the "AHAH cargo money goes -BRRRR"